test-lib.sh

```diff
@@ -32,30 +32,35 @@ UNSTABLE_TESTS="${UNSTABLE_TESTS:-""}"
 # Uses: $TESTSUITE_RESULT - overall result of all tests
 function ct_cleanup() {
   ct_show_resources
+  ct_show_results
+
+  if [[ -z ${CID_FILE_DIR:-} ]]; then
```

Comment:
Is this correct? I have missed that.

Reply:
Why you think this is not correct? Should I use `test` instead of `[[`?